makiflow/models/segmentation/utils.py

```diff
@@ -33,3 +33,43 @@ def mutate_masks(masks, mapping):
     return  new_masks
 
 
+def merging_masks(masks, index_of_main_mask, priority):
+    """
+    We choose the base mask which have index `index_of_main_mask` on which is put other classes
+    according to `priority` of other classes
+
+    Parameters
+    ----------
+    masks : list or numpy.array
+        List or numpy array of masks.
+    index_of_main_mask : int
+        Index of base mask.
+    priority : list
+        List of priority of classes. From highest priority to least priority.
+        Example: [9, 10, 3, 7], where 9 class have highest priority and will be put on others classes,
+        on the other hand the 7th class have the lowest priority and will be put on the others the latest.
```

Comment:
"will be put on the others the latest."
Will it be put first or last?
Logically, the class with the highest priority will be put as the last layer of the final mask.